### ISSUE TITLE
ACF - Änderungen

### ISSUE THREAD

**Issue by Forcase** [Forcase/wordpress-plugin_bigbluebutton#6]
Verstehe noch nicht, wie die ACFs mit dem BBB-PlugIn zusammenhängen. Was muss ich machen, sobald ich Änderungen bei den ACF vornehme. Zum Beispiel habe ich den Titel der ACF geändert und anschließend ein neues Feld hinzugefügt. Sowohl der Name wie auch das Feld werden nicht übernommen. Eine Neuinstallation des PlugIns hat nichts geändert. Kannst Du hier helfen?

**Comment by Forcase** [comment]
Habe gecheckt, dass das mit der class-bigbluebutton-admin.php zusammenhängt. Somit doppelter issue und meine Frage, ob wir das automatisieren können -> sprich wenn wir die ACF ändern hat das sofort Auswirkungen auf das PlugIn ohne den Code der PlugIns ändern zu müssen.

**Comment by lxbdr** [comment]
Nein, aber siehe meinen anderen Kommentar dann kannst du es ungestört zusammenbauen und ganz zum Schluss wird der Code wieder in das Plugin exportiert
